src/cp/vec.h

```diff
@@ -5,6 +5,16 @@
 #ifndef CPDEM_VEC_H
 #define CPDEM_VEC_H
 
+#define nearest_power_of_two(v)                                                                    \
+    (((v)--),                                                                                      \
+     ((v) |= (v) >> 1),                                                                            \
+     ((v) |= (v) >> 2),                                                                            \
+     ((v) |= (v) >> 4),                                                                            \
+     ((v) |= (v) >> 8),                                                                            \
```

Comment:
instead of reinventing the wheel, why not copying the code from rizin side?

Reply:
Yea, sorry about that 😅 I have a habit of doing that. The plus point is that `src/cp` will be a completely independent module on it's own if we just provide a `DemString` to it. Maybe that's good...?

Reply:
just define one like DemString (which is an extract of RzStrBuf) which any other demangler can use.